[{"name": "test.test_codon_to_prob.TestCodonToProb.test_content", "status": "failed", "message": "0.464 != 0.46 : Incorrect probability of codon UUU!", "passed": false, "points": ["p07-05.1"], "backtrace": ["  File \"/anaconda3/lib/python3.6/unittest/case.py\", line 59, in testPartExecutor\n    yield\n", "  File \"/anaconda3/lib/python3.6/unittest/case.py\", line 605, in run\n    testMethod()\n", "  File \"/Users/satu/Kurssit/hy-data-analysis-with-python-spring-2019/part07-e05_codon_to_prob/test/test_codon_to_prob.py\", line 26, in test_content\n    self.assertEqual(d[\"UUU\"], 0.46, msg=\"Incorrect probability of codon UUU!\")\n", "  File \"/anaconda3/lib/python3.6/unittest/case.py\", line 829, in assertEqual\n    assertion_func(first, second, msg=msg)\n", "  File \"/anaconda3/lib/python3.6/unittest/case.py\", line 822, in _baseAssertEqual\n    raise self.failureException(msg)\n"]}, {"name": "test.test_codon_to_prob.TestCodonToProb.test_size", "status": "passed", "message": "", "passed": true, "points": ["p07-05.1"], "backtrace": []}]